CONTRIBUTING.md

```diff
@@ -0,0 +1,60 @@
+# How to contribute
+
+We love getting patches to NServiceBus from our awesome community. Here is a few guidelines that we
+need contributors to follow so that we can have a chance of keeping on
+top of things.
+
+## Getting Started
+
+* Make sure you have a [GitHub account](https://github.com/signup/free)
+* [Create a new issue](https://github.com/NServiceBus/NServiceBus/issues/new), assuming one does not already exist.
+  * Clearly describe the issue including steps to reproduce when it is a bug.
+  * If it is a bug make sure you tell us what version you have encountered this bug on.
+* Fork the repository on GitHub
+
+## Running the Acceptance Tests
+ 
+Follow these steps to run the acceptance tests locally:
+
+* Add a new environment variable `Transport.UseSpecific` with the value `AzureServiceBusTransport`
+* Add a new environment variable `AzureServiceBusTransport.ConnectionString` containing a connection string to your Azure storage account 
+* Add a new environment variable `AzureServiceBusTransport.Topology` and set it to _ForwardingTopology_ to run tests configuring transport with `ForwardingTopology` topology. Don't setup `AzureServiceBusTransport.Topology` environment variable to run tests with `EndpointOrientedTopology` topology
+
+
+## Running Unit/Integration Tests
```

Comment:
I think this section is obsolete as well. At least we don't use a fallback connection string. The connection string name is wrong as well plus it looks like the tests are not using real connections, see for example https://github.com/Particular/NServiceBus.Transport.AzureServiceBus/blob/develop/src/Tests/Sending/MessageSenderPoolTests.cs#L12